build-tools/timing/timing.targets

```diff
@@ -92,7 +92,7 @@
         Condition=" '%(XACaptureBuildTimingProject.Restore)' != '' "
         Command="$(_TopDir).nuget\NuGet.exe restore &quot;%(XACaptureBuildTimingProject.Restore)&quot;"
     />
-    <Exec Command="$(_XABuild) /fl /flp:LogFile=$(_OutputFile).log &quot;/logger:$(_TimingLogger);ID=$(_ID);Description=$(_Description);Commit=$(XAVersionHash);OutputPath=$(_OutputFile).xml&quot; /p:Configuration=$(Configuration) /p:AndroidSupportedAbis=x86 %(XACaptureBuildTimingProject.Identity) /t:$(_Target)" />
+    <Exec Command="$(_XABuild) /fl /flp:LogFile=$(_OutputFile).log &quot;/logger:$(_TimingLogger);ID=$(_ID);Description=$(_Description);Commit=$(XAVersionHash);OutputPath=$(_OutputFile).xml&quot; /p:Configuration=$(Configuration) /p:AndroidSupportedAbis=x86 %(XACaptureBuildTimingProject.Identity) /t:$(_Target) /restore" />
```

Comment:
We need `/restore` *here* as well?

I believe what we *really* want is that *everywhere* we use `nuget restore` we should *also* call `msbuild /restore`, which would suggest:

```diff
diff --git a/build-tools/timing/timing.targets b/build-tools/timing/timing.targets
index e35c9c02..d06f26cc 100644
--- a/build-tools/timing/timing.targets
+++ b/build-tools/timing/timing.targets
@@ -92,6 +92,11 @@
         Condition=" '%(XACaptureBuildTimingProject.Restore)' != '' "
         Command="$(_TopDir).nuget\NuGet.exe restore &quot;%(XACaptureBuildTimingProject.Restore)&quot;"
     />
+    <MSBuild
+        Condition=" '%(XACaptureBuildTimingProject.Restore)' != '' "
+        Projects="%(XACaptureBuildTimingProject.Restore)"
+        Targets="Restore"
+    />
```